src/python_framework_v02/troute/nhd_io.py

```diff
@@ -180,8 +181,8 @@ def get_ql_from_wrf_hydro_mf(qlat_files, index_col="feature_id", value_col="q_la
     2018-01-01 13:00:00 4186117     41.233807 -75.413895   0.006496
     ```
     """
-    filter_list = None
 
+    filter_list = None
```

Comment:
I don't think we are using this parameter anywhere, so we can probably remove it.